pkg/smokescreen/conntrack/conn_tracker_test.go

```diff
@@ -84,8 +83,8 @@ func TestConnSuccessRateTracker(t *testing.T) {
 
 			sd := atomic.Value{}
 			sd.Store(false)
-			mockMetricsClient := &mockClientInterface{}
-			tracker := NewTracker(time.Second, &statsd.NoOpClient{}, logrus.New(), sd, StartNewConnSuccessRateTracker(500*time.Millisecond, 2*time.Second, 10*time.Second, mockMetricsClient))
+			mockMetricsClient := metrics.NewMockMetricsClient()
+			tracker := NewTracker(time.Second, metrics.NewNoOpMetricsClient(), logrus.New(), sd, StartNewConnSuccessRateTracker(500*time.Millisecond, 2*time.Second, 10*time.Second, mockMetricsClient))
```

Comment:
Nonblocking: I realize this is just a translation of the previous code, but this line is getting a bit unwieldy and it's a bit confusing to call `metrics.NewNoOpMetricsClient()` right after you create a `mockMetricsClient`. This line of code seems to use two metrics clients, and it's not obvious that that's intentional.

Reply:
Understand the confusion! I made the parameters multi-line and added a comment to explain why we're using a no-op here.